### ISSUE TITLE
Headings are not structured - Accordions

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#9663]
# Problem
## Headings are not structured - `S`

Heading tags help assistive technology users to understand how content is structured. They're also used for in-page screen reader navigation.

When adding heading tags, you should ensure that they follow a nested structure. h1 is used for the primary heading, followed by h2 for subheadings, and so on to h6.

Correcting your heading structure may also help to improve your site's SEO.

This is a best practice only and not currently required for WCAG conformance

- Help Center: [Headings are not structured, explained](https://help.siteimprove.com/support/solutions/articles/80001050742-sia-r53-headings-are-not-structured-explained)
- WCAG 2.2: [Technique H42: Using h1-h6 to identify headings](https://www.w3.org/WAI/WCAG22/Techniques/html/H42)
- Siteimprove Alfa: [Technical documentation for rule SIA-R53](https://alfa.siteimprove.com/rules/sia-r53)

# Possible Solution

I had submitted a ticket for this to Siteimprove 10 months ago at https://siteimprove.freshdesk.com/support/tickets/4292617.  

## Original report
https://my2.siteimprove.com/Inspector/998442/A11Y/Page?pageId=35290978067&impmd=167DAB416840EA5F51855348D3105222&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone&__hstc=177641658.9b104f177a52bbe7264c8e2ecfcc3766.1772741743818.1772741743818.1772741743818.1&__hssc=177641658.4.1772741743818&__hsfp=395a8584766fb0e85bc3ee217597d7cb#/sia-r53/failed/ajBNWi9IaWc=;NzY1MDYwYWQ=

## Submitted as false positive 4/23/25
https://my2.siteimprove.com/Accessibility/998442/NextGen/Activity/1/Title/Asc?tagId=2&pageSize=200&conformance=0,1,3,4&pageSegments=&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2&siteTargetIssueKinds=1,2&wcagVersion=22&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2. 

I will submit a ticket asking why this has taken so long.

**Comment by joewhitsitt** [comment]
# Grooming Conversation

**Quadrant:** 

Complex

**Discussion:**

- Seems to be a siteimprove issue based on our past choice to structure accordions using details
- At the time we wrote up a big document explaining why we made the switch to details and believe it was accessible
- "gotta be" an example of where either a details element properly highlights a heading or a perfectly accessible accordion structure
- one main reason we went with details was for the ctrl+f visibility into the content within so it would be nice to maintain that.
- possible research/investigation spike into whether we should switch off details again for siteimprove a11y nirvana.

**Proposed solution or definition of done:**

- Housed within this issue, the research spike should remain here and any follow-up implementation piece should be a separate slice, planned and prioritized.
- Investigate how to make all the things work with accordions
